superset/assets/src/dashboard/util/getFilterConfigsFromFormdata.js

```diff
@@ -31,9 +31,15 @@ export default function getFilterConfigsFromFormdata(form_data = {}) {
   } = form_data;
   let configs = filter_configs.reduce(
     ({ columns, labels }, config) => {
+      let defaultValues = config.defaultValue;
+      // defaultValue could be ; separated values,
+      // could be null or ''
+      if (config.defaultValue) {
+        defaultValues = config.defaultValue.split(';');
```

Comment:
defaultValues is now always an array, even if there's only one element in it. is that an issue?

Reply:
Yeah, this is how filter_box component handle initial values:
https://github.com/apache/incubator-superset/blob/a96eae46af852ba22ba1f2427333bc7d03ebd6c1/superset/assets/src/visualizations/FilterBox/FilterBox.jsx#L241

from the above line, i feel array is preferred type.